src/backend/postmaster/pgstat.c

```diff
@@ -2913,6 +2915,10 @@ gpstat_report_waiting(char reason)
 	 * protocol.  The update must appear atomic in any case.
```

Comment:
It seems that this comment isn't actual after this patch. Why didn't you wrap changes with the st_changecount protocol?

Reply:
Fixed. Thanks.